core/src/test/java/org/bitcoinj/wallet/DeterministicKeyChainTest.java

```diff
@@ -470,6 +475,180 @@ public void watchingChainAccountOne() throws UnreadableWalletException {
         assertEquals(key4.getPubKeyPoint(), rekey4.getPubKeyPoint());
     }
 
+    @Test
+    public void spendingChain() throws UnreadableWalletException {
+        Utils.setMockClock();
+        DeterministicKey key1 = chain.getKey(KeyChain.KeyPurpose.RECEIVE_FUNDS);
```

Comment:
avoid variables ending in numbers. If those are to be treated equally, instantiate a list/array. If not, name them by their purpose. Makes review easier.

In particular: Are key1 and key2 supposed to differ or is key2 a copy of key1?

(Granted, this is the same style as other tests in the file.)

Reply:
I will rename the key1 as firstReceiveKey, key2 as secondReceiveKey, key3 as firstChangeKey and key4 as secondChangeKey.  These are all different keys for the first two receive addresses and first two change addresses in the key chain.